src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -255,13 +262,6 @@ public boolean isFLAC() {
 	}
 
 	/**
-	 * @return True if the audio codec is HE-AAC.
-	 */
-	public boolean isHEAAC() {
-		return FormatConfiguration.AAC_HE.equalsIgnoreCase(getCodecA());
```

Comment:
Lol, now I understand. It seems we've had some previous merge conflict resolution here with side effects. Sorry for being slow...